### PR TITLE
feat(recording) allow highlighting meeting recording moments

### PR DESCRIPTION
**Pull request by quitrk**

### CODE REVIEW

File: react/features/base/icons/svg/bookmark.svg [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path fill-rule="evenodd" clip-rule="evenodd" d="M6 5V17.8658L10.1379 14.59C11.229 13.7261 12.771 13.7261 13.8621 14.59L18 17.8658V5H6ZM5 3C4.44772 3 4 3.44771 4 4V19.9329C4 20.769 4.96513 21.2359 5.6207 20.7169L11.3793 16.1581C11.743 15.8701 12.257 15.8701 12.6207 16.1581L18.3793 20.7169C19.0349 21.2359 20 20.769 20 19.9329V4C20 3.44772 19.5523 3 19 3H5Z" fill="#C2C2C2"/>
```
Comment (saghul) [inline react/features/base/icons/svg/bookmark.svg:2]: Don't set the fill in the icon please.

**Comment by quitrk** [comment]
jenkins please test this please

File: react/features/recording/components/Recording/AbstractHighlightButton.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+ * @param {Object} state - The Redux state.
+ * @private
+ * @returns {{
+ *     _audioOnly: boolean
+ * }}
+ */
+export function _abstractMapStateToProps(state: Object) {
+    const isRecordingRunning = getActiveSession(state, JitsiRecordingConstants.mode.FILE);
+    const isButtonDisabled = isHighlightMeetingMomentDisabled(state);
+    const { webhookProxyUrl } = state['features/base/config'];
```
Comment (saghul) [inline react/features/recording/components/Recording/AbstractHighlightButton.js:65]: Oh, this one is not documented in config.js (not your fault)

**Review by saghul — APPROVED** [review]
LGTM. What about the mobile side?

**Comment by quitrk** [comment]
> LGTM. What about the mobile side?

coming in a future PR

**Pull request merged**